ccd-definition/ComplexTypes/AllocatedJudge.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "AllocatedJudge",
```

Comment:
I think we could reuse JudgeAndLegalAdvisor and then hide LA in CaseEventToComplexTypes...
but maybe it is better to have a separate model for AllocatedJudge like this (otherwise we have a JudgeAndLegalAdvisor which never has a legal advisor...), fine either way, maybe @swalker125 opinion if you're reviewing

Reply:
Yes wasn't sure on this one lets see what @swalker125 thinks

Reply:
I find our implementation of Judge and Legal Advisor really confusing. We capture this information lots of times and I am confused about which entry relates to which thing 😞 It would be good to reuse - what do you think the effort is @toireasam

Reply:
Can give this a go in CaseEventToComplexTypes as Joe mentioned - reason i had it separated was because this event is mandatory and doesn't include the legal advisor but i can try to reuse and comment back on this how i've got on. :)

Reply:
this is changed now :)